components/channel_layout/channel_controller.tsx

```diff
@@ -38,6 +51,16 @@ export default function ChannelController({shouldShowAppBar, fetchingChannels}:
         };
     }, []);
 
+    useEffect(() => {
+        const loadStatusesIntervalId = setInterval(() => {
+            dispatch(loadStatusesForChannelAndSidebar());
+        }, Constants.STATUS_INTERVAL);
```

Comment:
this was inside of team controller before, But logically this component is a better place for it